reco_utils/recommender/tfidf/tfidf_utils.py

```diff
@@ -217,10 +217,10 @@ def get_tokens(self):
         return self.tokens
 
     def get_stop_words(self):
-        """ Return the stop words excluded in the TF-IDF vectorizer.
+        """Return the stop words excluded in the TF-IDF vectorizer.
 
         Returns:
-            frozenset: Frozenset of stop words used by the TF-IDF vectorizer (can be converted to list).
+            obj: Frozenset of stop words used by the TF-IDF vectorizer (can be converted to list).
```

Comment:
Again `object`

Reply:
Actually this should be `list or None` according to scikit-learn documentation.

Reply:
changed to list